mediator/med.F90

```diff
@@ -948,7 +823,7 @@ subroutine InitializeIPDv03p1(gcomp, importState, exportState, clock, rc)
     do ncomp = 1,ncomps
        if (ncomp /= compmed) then
           if (mastertask) write(logunit,*)
-          if (ESMF_StateIsCreated(is_local%wrap%NStateImp(ncomp))) then
+          !if (ESMF_StateIsCreated(is_local%wrap%NStateImp(ncomp))) then
```

Comment:
If these checks no longer needed, remove the commented out lines.

Reply:
Fixed